.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,32 @@
-### Context
+## Ticket and context
 
-### Changes proposed in this pull request
+Ticket:
 
-### Guidance to review
+## Code review
```

Comment:
```suggestion
## Review
```

IMO we don't just review code in a PR; we want to check that it actually works as intended.